bittensor/subtensor.py

```diff
@@ -4415,7 +4415,7 @@ def make_substrate_call_with_retry():
             result = make_substrate_call_with_retry()
         except scalecodec.exceptions.RemainingScaleBytesNotEmptyException:
             bittensor.logging.error(
-                "Your wallet it legacy formatted, you need to run btcli stake --ammount 0 to reformat it."
+                "Received a corrupted message. This likely points to an error with the network or subnet."
```

Comment:
Maybe we can add the legacy wallet info to your new messages as well.  Like "...likely points to an error with the network/subnet, or may be due to usage of a legacy-formatted wallet."    Open to ideas here. Not sure how many legacy wallets there are floating out there